demos/src/scss/_demos.scss

```diff
@@ -48,10 +48,9 @@ body:before {
 }
 
 .o-grid-container {
-	background: lighten(#f6e9d8, 8.5%);
+	background: #EAE8DF;
 }
```

Comment:
How about a documented `$o-grid-debug` variable set manually to false in the demos, that if true activates the gutter colouring?